e2e/tests/121-Portal/ViewPaProfilePage/ViewActivityOverviewMessageTab.spec.ts

```diff
@@ -8,6 +8,7 @@ import { seedPaidRegistrations } from '@121-service/test/helpers/registration.he
 import { resetDB } from '@121-service/test/helpers/utility.helper';
 import { registrationsOCW } from '@121-service/test/registrations/pagination/pagination-data';
 import { test } from '@playwright/test';
+import data from '../../../../../121-platform/interfaces/Portal/src/assets/i18n/en.json';
```

Comment:
Looks like you're importing the same thing twice

Reply:
Fixed